BibTeX.js

```diff
@@ -262,7 +263,14 @@ function setKeywordDelimRe( val, flags ) {
 function processField(item, field, value, rawValue) {
 	if(Zotero.Utilities.trim(value) == '') return null;
 	if(fieldMap[field]) {
-		item[fieldMap[field]] = value;
+		//map DOIs + Label to Extra for unsupported item types
+		if (field == "doi" && item.itemType != "journalArticle" && item.itemType != "conferencePaper") {
```

Comment:
I like the (more general) function `ZU.fieldIsValidForType("DOI", item.itemType)` which you used in another translator. How about using it here as well?